src/DynamoCore/Library/FunctionDescriptor.cs

```diff
@@ -170,7 +170,15 @@ public string Summary
         /// </summary>
         public string Description
         {
-            get { return !String.IsNullOrEmpty(Summary) ? Summary : string.Empty; }
+            get { return !String.IsNullOrEmpty(Summary) ? Summary + "\n\n" + Signature : Signature; }
+        }
+
+        /// <summary>
+        ///     A comment describing the function
+        /// </summary>
+        public string LibraryDescription
+        {
+            get { return String.IsNullOrEmpty(Summary) ? string.Empty : Summary; }
```

Comment:
Hi @aosyatnik, I'm expecting the `DynamoToolTip` inside `NodeView.xaml` to be changed -- add another `TextBlock` for `Signature` property below `TextBlock` for `Description`. Is that possible? I don't think we should change at this level.